panther_analysis_tool/main.py

```diff
@@ -507,6 +514,15 @@ def classify_analysis(
                 analysis_ids.append(analysis_spec['RuleID'])
                 analysis.append(
                     (analysis_spec_filename, dir_name, analysis_spec))
+            if analysis_spec['AnalysisType'] == 'scheduled_query':
+                keys = list(SCHEDULED_QUERY_SCHEMA.schema.keys())
+                SCHEDULED_QUERY_SCHEMA.validate(analysis_spec)
+                if analysis_spec['QueryName'] in analysis_ids:
+                    raise AnalysisIDConflictException(
+                        analysis_spec['QueryName'])
+                analysis_ids.append(analysis_spec['QueryName'])
+                queries.append(
+                    (analysis_spec_filename, dir_name, analysis_spec))
```

Comment:
I'm realizing this might have some pretty fun merge conflicts with #76 :-/

Reply:
Luckily #76 seems to have cleaned things up considerably. I essentially just threw my changes away here and re-did them with little issue, will bring this out of draft today.